include/swift/IDE/Utils.h

```diff
@@ -390,7 +399,8 @@ class NameMatcher: public ASTWalker {
   bool shouldWalkAccessorsTheOldWay() override { return true; }
 
 public:
-  explicit NameMatcher(SourceFile &SrcFile) : SrcFile(SrcFile) { }
+  explicit NameMatcher(SourceFile &SrcFile, bool ForExtArgName = false)
+      : SrcFile(SrcFile), ForExtArgName(ForExtArgName) {}
```

Comment:
I’m not a fan of adding a parameter here. And: Why is this even necessary. Shouldn’t the locations in `UnresolvedLoc` be sufficient to determine whether we are referring to the external or internal parameter name, just based on the location?